app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningFragment.kt

```diff
@@ -37,6 +39,14 @@ class LearningFragment : Fragment() {
 
     lateinit var learningPresenter: LearningPresenter
 
+    /**
+     * model for evaluation mode
+     * A null value represents the fact that the learning mode is not EVALUATION
+     *
+     * TODO ideally the model would be wrapped inside the presenter (but no time right now)
+     */
+    private var evaluationModel: EvaluationModel? = null
```

Comment:
Would it be cleaner to use something like an option in scala? (If there's such a thing in kotlin)